### DISCUSSION TITLE
F1 data analysis

### DISCUSSION THREAD

**Discussion by PrabhuPugal** [theOehrly/Fast-F1#596] [Q&A]
Yo I'm planning to start a data analysis project using fastf1 api. I really don't know how the data is extracted and in what form the data is being stored. I have many doubts and I'm an intermediate in programming and trying to use python. I don't know where to start, need help.

**Comment by Casper-Guo** [comment]
What do you want to work on? Have you checked out the example gallery in the docs?

**Reply by PrabhuPugal** [reply]
I wanna pull the sessional data, for example tyre degradation for the current track temp and suggest on which lap will it be ideal to do a pitstop

**Reply by Casper-Guo** [reply]
Accessing lap time and compound type data, from which you can infer tyre degradation, is trivial. See [this example](https://docs.fastf1.dev/examples_gallery/plot_driver_laptimes.html#sphx-glr-examples-gallery-plot-driver-laptimes-py).

Unfortunately there isn't any example working with weather data now but you can check out [this section](https://docs.fastf1.dev/core.html#fastf1.core.Laps.get_weather_data) of the docs.
